static/inline-spotlight/js/vendor/jquery.ellipsis.js

```diff
@@ -0,0 +1,101 @@
+/*! jQuery ellipsis - v1.1.1 - 2014-02-23
+* https://github.com/STAR-ZERO/jquery-ellipsis
+* Copyright (c) 2014 Kenji Abe; Licensed MIT */
```

Comment:
hmm yeah , but the `trimText` it's a little bit different, it cuts text and the ellispsis adapts the contents to the container.